### PR TITLE
Embedding images and video

### PR DESCRIPTION
**Pull request by jstloyal**
Created a youtube like website with video and some images embedded in it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
-# youtube_video_page
\ No newline at end of file
+# youtube_video_page
+
+> Replica of Youtube Video Page (Microverse: Embedding Images and Video)
+
+![screenshot](./app_screenshot.png)
```
Comment (euqueme) [inline README.md]: There is no such image called **app_screenshot.png** in your project, please upload the image or remove it from the Readme file

File: README.md [hunk 2]
```diff
@@ -1 +1,69 @@
...
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```
Comment (euqueme) [inline README.md]: Add information to each title from line #25 to line #35 or delete them from your Readme file

File: README.md [hunk 3]
```diff
@@ -1 +1,69 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- [Informit sidebar code](http://www.informit.com/articles/article.aspx?p=1807656&seqNum=8)
+- Inspiration
+- etc
```
Comment (euqueme) [inline README.md:32]: remove `- etc` on line #65

File: README.md [hunk 4]
```diff
@@ -1 +1,69 @@
...
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
```
Comment (euqueme) [inline README.md]: specify the steps or remove line #23 from your Readme file

File: README.md [hunk 5]
```diff
@@ -1 +1,69 @@
...
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```
Comment (euqueme) [inline README.md]: delete the instructions to use the Readme template (line #20 and #21)

File: README.md [hunk 6]
```diff
@@ -1 +1,69 @@
-# youtube_video_page
\ No newline at end of file
+# youtube_video_page
+
+> Replica of Youtube Video Page (Microverse: Embedding Images and Video)
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
```
Comment (euqueme) [inline README.md]: delete this line #7

File: index.html [hunk 7]
```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Contributing to Open Source Part 1</title>
+    <link href="./css/main.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <div class="nav">
+      <div class="image-logo">
+        <img src="img/youtubelogo.jpeg" style="width: 100px;" />
```
Comment (euqueme) [inline index.html]: all the styling should be declared in the CSS file, remove the style for your image on line #10

File: index.html [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+          <input type="text" placeholder="Search..." />
+          <button type="submit">Submit</button>
+        </form>
+      </div>
+
+      <div class="image-group">
+        <figure>
+          <img src="img/video.png" alt="Vedoes" style="width: 50px;" />
+          <img src="img/apps.png" alt="google apps" style="width: 50px;" />
+          <img src="img/notify.jpeg" alt="Notifications" style="width: 50px;" />
```
Comment (euqueme) [inline index.html]: all the styling should be declared in the CSS file, remove the style for your images on lines #22, #23 and #24

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @jstloyal  and @harshdeepkanhai :balloon:  :lollipop: 

### The effort you put in your project is remarkable :muscle: but there is still some work to do to make it look more like the original :construction: :construction_worker_man: 

## Page Structure

**Navbar**

- [x] The navbar should be smaller, all the icons in the navbar should be smaller including the logo, the search bar should be wider and the background color should be white.

- Yours

image

- Original

image

**Video**

* [x] It shows the number of views
* [x] It has like and dislike icons (yours is missing the dislike button)
* [x] the Share should be a link or a button and the background should be white
* [x] It shows the avatar of the user who uploaded the video (The Avatar should be an image)
* [x] It has a subscribe button (the Subscribe should be a red button)

- Yours

image

- Original

image

**Sidebar Recommended videos**

* [x] It shows the title of the video ( Please put titles of real videos don't use meaningless text unless is specified in the project description)  
* [x] It shows the number of views (put an actual number of views don't use meaningless text unless is specified in the project description) 
* [x]  the title and views have to be at the right of each thumbnail and not below them

- Yours

image

- Original

image

## Generals

**Readme**

- [x]  remove all the template instructions and also remove the empty sections or  add info to them, I have put comments on your code where changes should be made :arrow_double_down: 

**Inline styling**

- [x] remove all the inline styling I have spotted in your code as all styling should live in the CSS file :arrow_double_down: 

[OPTIONAL] I strongly encourage the use of the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links (navbar)

### That's it! :raised_hands: 

### Please :pray:  do the changes :memo:  and submit back for a code review when you are done :rocket: 

Happy Coding! :woman_technologist: 

Any question? reach to me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: 

Hi @jstloyal and @harshdeepkanhai you have done a great job 👍 Making changes requested by previous reviewer. 🙂 
- You made changes to the video section 👍 But the subscribe button is currently not a button. Kindly fix it
[OPTIONAL]
Currently, the sidebar does not look good. We can make it look better 😄 

Kindly make these changes and submit another code review 🎈 
Happy coding 💻 
Daniel Larbi Addo

**Comment by jstloyal** [comment]
I've made the changes!

**Review by euqueme — COMMENTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @jstloyal and @harshdeepkanhai  :handshake: 

### Thank you for doing the changes requested by me and the previous reviewer :medal_sports: 

### I have only one change to ask you for the project to be approved because the previous reviewer didn't mention some changes as a follow-up of my review, I'm mentioning them bellow but setting them as optional :point_down: 

## General

- [ ] please fix the HTML errors and warnings detected by the validator as we can't approve projects that have HTML errors

image

## Video

- [ ] [OPTIONAL] the video has to be closer to the sidebar actually, it should be bigger in size, look at the original

- Yours 

image

- Original

image

## Sidebar Recommended Videos

- [ ] [OPTIONAL] the views should be aligned to the left, some of them are shifting, also add some margin on the right of this section because the description of the last video looks like its right in the border of the screen

- Yours

image

- Original

image

### That's it, team! :raised_hands: 

### Please :pray:  do the changes :memo: and submit back for a code review when you are done :rocket: 

Happy coding! :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES  :traffic_light:

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — COMMENTED** [review]
## Status: Require changes 🔴

Hi @jstloyal and @harshdeepkanhai, nice work 👍 . For your project to be approved you need to meet the minimum requirements. Currently, we are failing markup errors as requested by the previous reviewer 🙂 .
ma
 Kindly fix it and submit code review
[OPTIONAL]
```Video Section``` does not look good. There is no spacing between the main video and the video thumbnail. Check image for clarification
ma1

Happy coding 💻 
Daniel Larbi Addo

**Review by pelzolga123 — COMMENTED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Hi Team,
## Approved, Congratulation :+1: 
You did all changes required previously. Go ahead and merge to master
_Goodluck in your next project_ *Happy coding*

**Pull request merged**